admin/common_auth/models.py

```diff
@@ -73,7 +73,14 @@ class Meta:
     #     return self.is_admin
 
     @property
+    def full_name(self):
```

Comment:
There is traditionally a method on the django user model called `get_full_name` it might be worth using that instead of having this property or having the property call that method and changing what that method returns. Most django admin UI things use `get_full_name` or `get_short_name` to display info on a user.